app/assets/javascripts/app/collections/aspect_selections.js

```diff
@@ -0,0 +1,30 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
+
+app.collections.AspectSelections = Backbone.Collection.extend({
+  model: app.models.AspectSelection,
+
+  selectedAspects: function(attribute) {
```

Comment:
I don't really feel we need the "Aspects" word in that name, because it is in the `AspectSelections` class. `selected` here implicitly means that it is about apsects. And since the method gets attribute from selected aspects, I would call it `getSelectedAttribute` or `selectedGetAttribute` or just `selectedAttribute`.